app/models/concerns/account_followable.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module AccountFollowable
```

Comment:
Instead of this being a concern, could the whole thing be a new class?

Something like `FollowableFilter.new(account).accounts` as the scope in the class?

Reply:
It is also only ever used by `AccountSuggestions::GlobalSource#scope` so maybe it could be moved to this class.

Reply:
It's also in `AccountSuggestions::SettingSource`, but they both inherit from same base class and use it in similar ways ... will look at moving in there as well.